addons/web/static/src/js/views/control_panel/control_panel_model.js

```diff
@@ -251,7 +251,7 @@ var ControlPanelModel = mvc.Model.extend({
             facets: facets,
```

Comment:
[FIX] web: do not use groupbys in non groupable views

Before this commit: Some problems occur with non groupable views (views for which searchMenuTypes does not contain 'groupBy'):
- The searchbar still displays autocomplete suggestions for groupby filters defined in the search arch. Selecting one of them will cause a crash. 
- If the action has a default groupby (set in Studio for instance) then it is still applied and causes crashes.
Those problems happen in the documents kanban view for instance.

After this commit: If a view is not groupable,
- The groupby filters coming from the search arch are not displayed at all in the interface,
- A default groupby is not considered.

Reply:
@Polymorphe57 👍